### PR TITLE
Create inactive_users_for_integrated_apps.pt

### PR DESCRIPTION
**Pull request by nia-vf1**
Created policy for Managed SaaS Applications which have Integrated Applications with Inactive Users

### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: saas/fsm/inactive_users_for_integrated_apps/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+- *Email addresses to notify* - Email addresses of the recipients you wish to notify
+- *Application Name* - Name of the managed application for which you wish to retrieve inactive users for its integrated applications.
+
+## Prerequisites
+
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
```
Comment (rshade) [inline saas/fsm/inactive_users_for_integrated_apps/README.md:23]: Can we update this one to the flexera, this has some examples: https://github.com/flexera-public/policy_templates/pull/884

File: saas/fsm/inactive_users_for_integrated_apps/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Other prerequisites:
+
+- Ensure that the Application integration task is successfully configured and executing its application discovery task for the Managed SaaS Application.
+
+### Credential configuration
+
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed:
```
Comment (rshade) [inline saas/fsm/inactive_users_for_integrated_apps/README.md:31]: This one also

**Review by rshade — CHANGES_REQUESTED** [review]

File: saas/fsm/inactive_users_for_integrated_apps/README.md (hunk 1 above)
Comment (nia-vf1) [reply saas/fsm/inactive_users_for_integrated_apps/README.md]: Updated

File: saas/fsm/inactive_users_for_integrated_apps/README.md (hunk 2 above)
Comment (nia-vf1) [reply saas/fsm/inactive_users_for_integrated_apps/README.md]: updated

**Review by rshade — APPROVED** [review]
LGTM

**Pull request merged**